lib/src/rules/prefer_contains.dart

```diff
@@ -193,4 +174,12 @@ class _Visitor extends SimpleAstVisitor<void> {
 
     return true;
   }
+
+  static TokenType _invertedTokenType(TokenType type) => switch (type) {
```

Comment:
This looks so nice!

Reply:
Bonus points for being generated by our assist. 🥳

Reply:
/fyi @munificent

Reply:
For points, you could make this an `extension on TokenType` 😄

Reply:
https://github.com/dart-lang/linter/pull/4272